pxd.c

```diff
@@ -251,7 +251,6 @@ static struct fuse_req *pxd_fuse_req(struct pxd_device *pxd_dev)
 			 __func__, status);
 	}
 
-	req->fastpath = pxd_dev->fastpath;
```

Comment:
Can we not just check if req is valid and update this condition. I was relying if fuse req is available, then its fastpath state is always in sync. Actually should check for IS_ERR_OR_NULL to be safe as below.

if (!IS_ERR_OR_NULL(req)) {
   req->fastpath = pxd_dev->fastpath;
}

And this implies not changing anywhere else.

Reply:
pxd_fuse_req() just does one thing: allocate request. Which i think is the right way.
The initialization is at the caller.

Reply:
Ok, in that case, I see only 2 places as you had fixed up for fastpath init. 
But the condition for failed fuse req allocation should still change to IS_ERR_OR_NULL.

Reply:
Right, done.